.github/workflows/add-discourse-topic.yml

```diff
@@ -0,0 +1,30 @@
+name: Add discourse topic
+on:
+  workflow_dispatch:
+jobs:
+  add-discourse-topic:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Get the branch
+      - name: Get branch
+        uses: actions/checkout@v3
+
+      # Identify the action
+      - name: Identify the action
+        run: |
+          git config user.name "GitHub Action (Add discourse topic)"
+          git config user.email "<support@dockstore.org>"
```

Comment:
Cool, I see that `dockstore-bot` was making commits in this PR. Is this the config info for `dockstore-bot`? (curious)

Reply:
Looks like the above information is stored with the commit in the repo itself (displayed via `git log`), but that github displays the commit as authored by `dockstore-bot`.  That's probably ok, we could try to make them match, but I like the detailed info in the log.